addons/account/report/account_invoice_report.py

```diff
@@ -44,7 +44,8 @@ class AccountInvoiceReport(models.Model):
     product_categ_id = fields.Many2one('product.category', string='Product Category', readonly=True)
     invoice_date_due = fields.Date(string='Due Date', readonly=True)
     account_id = fields.Many2one('account.account', string='Revenue/Expense Account', readonly=True, domain=[('deprecated', '=', False)])
-    price_subtotal = fields.Float(string='Untaxed Total', readonly=True)
+    price_subtotal_currency = fields.Float(string='Untaxed Amount in Currency', readonly=True)
```

Comment:
Little advice easy to remember if you hesitate on double or single quotes, all the parameter that will be displayed to the users (for example the string) should be in double quotes, i know the other field in the file use single quote so i will let you decide if you just want to be consistent with what has been done before :smile:

Reply:
> Little advice easy to remember if you hesitate on double or single quotes, all the parameter that will be displayed to the users (for example the string) should be in double quotes, i know the other field in the file use single quote so i will let you decide if you just want to be consistent with what has been done before 😄

Thanks for pointing that out, will take care of that afterwards!
I always prefer to maintain consistency with the existing style in a file to avoid confusion other developers who may work on it later on.